donut/modules/courses/utils/import_registrar.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+# Example command: ./import_registrar.py donut_courses_FA2018-19_180515.txt 2018 FA
```

Comment:
Maybe also document procedure for loading in courses in wiki. Also, thinking about the above, maybe we wouldn't want to clean out the databases, but definitely the source files? for the former, it would probably be useful for additional features like ratemyprof, course recommender, etc.

Reply:
One of the goals of the way I designed the database was to make it so you never have to remove old courses. In theory, you should be able to throw away the courses file as soon as it's imported. I didn't want to force people to put the files in any specific place, so I figured it made sense for the user to always give a full path to the file.